### ISSUE TITLE
Sentry: code sample uses a deprecated member

### ISSUE THREAD

**Issue by miksen** [flutter/flutter#57896]
The [Sentry package](https://pub.dev/packages/sentry) contains this recommended code for catching general errors:

```dart
var sentry = SentryClient(dsn: "https://...");
// Run the whole app in a zone to capture all uncaught errors.
runZoned(
  () => runApp(MyApp()),
  onError: (Object error, StackTrace stackTrace) {
    try {
      sentry.captureException(
        exception: error,
        stackTrace: stackTrace,
      );
      print('Error sent to sentry.io: $error');
    } catch (e) {
      print('Sending report to sentry.io failed: $e');
      print('Original error: $error');
    }
  },
);
```
Using this code in an app, results in this analysis warning about a deprecated member:

```
'onError' is deprecated and shouldn't be used. Use runZonedGuarded instead.
Try replacing the use of the deprecated member with the replacement.
```

## Version info

```
$ flutter --version
Flutter 1.17.1 • channel stable • https://github.com/flutter/flutter.git
Framework • revision f7a6a7906b (12 days ago) • 2020-05-12 18:39:00 -0700
Engine • revision 6bc433c6b6
Tools • Dart 2.8.2
```

**Comment by miksen** [comment]
Looks like this was fixed in https://github.com/flutter/sentry/pull/55, but a new version wasn't published to pub.dev?

**Comment by iapicca** [comment]
Hi @miksen 
can you please provide your `pubspec.yaml` 
Thank you

**Comment by miksen** [comment]
This repros with a new app from `flutter create` from 1.17.1 stable channel.

**Comment by TahaTesser** [comment]
Hi @miksen 
Looks like this merged and should be available sentry 4.0, current version is 3.0.1
Closing, as this isn't an issue with Flutter itself, if you disagree please write in the comments and I will reopen it 
Thank you
